Find a functionally equivalent code:android.animation.AnimatorSet.playSequentially:COMMENT
Method Modifier: public      
Comment:/**
 * Sets up this AnimatorSet to play each of the supplied animations when the
 * previous animation ends.
 *
 * @param items The animations that will be started one after another.
 */

@@ -1,11 +1,12 @@
 {
-    if (items != null && items.size() > 0) {
+    if (items != null) {
         mNeedsSort = true;
-        if (items.size() == 1) {
-            play(items.get(0));
+        if (items.length == 1) {
+            play(items[0]);
         } else {
-            for (int i = 0; i < items.size() - 1; ++i) {
-                play(items.get(i)).before(items.get(i + 1));
+            mReversible = false;
+            for (int i = 0; i < items.length - 1; ++i) {
+                play(items[i]).before(items[i + 1]);
             }
         }
     }

